package/gluon-web-osm/Makefile

```diff
@@ -8,4 +8,14 @@ define Package/gluon-web-osm
   TITLE:=base src for OSM inlay
 endef
 
+define Package/gluon-config-mode-core/install
+       $(Gluon/Build/Install)
+
+ifdef CONFIG_GLUON_MINIFY
```

Comment:
I think it would be a good idea to add `PKG_CONFIG_DEPENDS += CONFIG_GLUON_MINIFY` to `gluon-web-osm` and `gluon-web-model`. While it's added implicitly by `gluon.mk` due to the existence of a `luasrc` directory, it seems clearer to add it explicitly, as it isn't only used for Lua in these packages now.

Reply:
resolved in ed3551a5ca734556f861f4c62ffb199f2830ad44 and 1ef6daafdff3133314dcb6a38fed2a0f9e93c3f6